package.json

```diff
@@ -2,7 +2,10 @@
   "name": "something-awesome",
   "version": "1.0.0",
   "description": "test for submitting pull requests in command line",
-  "main": "index.js",
+  "main": "app.js",
```

Comment:
Good call changing this, but it should also include the path. So it should be `src/app.js`.